.travis.yml

```diff
@@ -1,17 +0,0 @@
-language: ruby
```

Comment:
this seemed less than useful to stick around given we don't run the main repo CI